### PR TITLE
feat: Supports verification of multiple WebAuthn enrollments

### PR DESCRIPTION
**Pull request by gowthamidommety-okta**
Resolves: OKTA-229243

multiplewebauthnverify

### CODE REVIEW

File: test/unit/helpers/xhr/MFA_CHALLENGE_multipleWebauthn.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+      "challenge": {challenge: "kygOUtSWURMv_t_Gj71Y", extensions: {"appid": "https://foo.com"}},
+      "factors": [{
+        "id": "webauthnFactorId1",
+        "factorType": "webauthn",
+        "provider": "FIDO",
+        "vendorName": "FIDO",
+        "profile": {"credentialId": "vdCxImCygaKmXS3S_2WwgqF1LLZ4i_2MKYfAbrNByJOOmSyRD_STj6VfhLQsLdLrIdgvdP5EmO1n9Tuw5BawZw"},
+        "_embedded": {
+          "challenge": {
+            "challenge": "abcd",
```
Comment (mauriciocastillosilva-okta) [inline test/unit/helpers/xhr/MFA_CHALLENGE_multipleWebauthn.js]: nit: `challenge1` and rest the same, similar to your ids `challenge2` and `challenge3`

File: test/unit/helpers/xhr/MFA_REQUIRED_multipleWebauthn.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        "href": "https:\/\/foo.com\/api\/v1\/authn\/cancel",
+        "hints": {
+          "allow": [
+            "POST"
+          ]
+        }
+      }
+    }
+  }
+});
```
Comment (mauriciocastillosilva-okta) [inline test/unit/helpers/xhr/MFA_REQUIRED_multipleWebauthn.js]: nit: new line

File: test/unit/spec/VerifyWebauthn_spec.js [hunk 3]
```diff
@@ -149,47 +212,168 @@ function (Okta,
...
+      return setupFn({webauthnSupported: true}).then(function (test) {
         expect(test.form.el('o-form-error-html')).toHaveLength(0);
       });
     });
 
     itp('shows a spinner while waiting for webauthn challenge', function () {
-      return setupWebauthnFactor({webauthnSupported: true}).then(function (test) {
+      return setupFn({webauthnSupported: true}).then(function (test) {
         expect(test.form.el('webauthn-waiting').length).toBe(1);
+        return Expect.waitForSpyCall(webauthn.isNewApiAvailable);
```
Comment (mauriciocastillosilva-okta) [inline test/unit/spec/VerifyWebauthn_spec.js]: you don't need this `Except` right? the test ends here.

Comment (gowthamidommety-okta) [reply test/unit/spec/VerifyWebauthn_spec.js]: You're right sorry! Must've added by mistake when refactoring! Thanks for catching this!!!

**Pull request merged**